cpp/examples/deck.gl/flight-paths.cc

```diff
@@ -31,13 +31,13 @@ using namespace deckgl;
 loadersgl::JSONLoader jsonLoader;
 auto fileSystem = std::make_shared<arrow::fs::LocalFileSystem>();
 
-auto createInitialViewState() -> std::shared_ptr<ViewState> {
+auto createViewState(double bearing) -> std::shared_ptr<ViewState> {
   auto initialViewState = std::make_shared<ViewState>();
   initialViewState->latitude = 47.65;
   initialViewState->longitude = 7.0;
   initialViewState->zoom = 4.5;
   initialViewState->pitch = 50.0;
-  initialViewState->bearing = 0.0;
+  initialViewState->bearing = bearing;
```

Comment:
If we are changing it we should not call it initialViewState.

We don't have interactivity and controllers implemented but the semantics of initialViewState is that deck will internally track a viewState, where as if we pass in the viewState prop, the app takes control of the viewState.